Docs/source/running_cpp/parameters.rst

```diff
@@ -386,6 +386,10 @@ Particle initialization
     It only works if `<species>.do_qed = 1`. Enables non-linear Breit-Wheeler process for this species.
     **Implementation of this feature is in progress. It requires to compile with QED=TRUE**
 
+* ``E_external & B_external`` (list of `float`) optional (default `0.0`)
```

Comment:
Could you split the two options in this script and add the `warpx` prefix, e.g., replace
```
``E_external & B_external`` (list of `float`) optional (default `0.0`)
```
with
```
``warpx.E_external`` & ``warpx.B_external`` (list of `float`) optional (default `0.0`)
```

Reply:
done